vmdb/app/models/miq_request_workflow.rb

```diff
@@ -116,12 +116,12 @@ def init_from_dialog(values, userid)
 
     self.get_all_dialogs.keys.each do |d|                         # Go thru all dialogs
       self.get_all_fields(d).keys.each do |f|                     # Go thru all field
-        unless options[f].nil?
+        if options[f]
           values_new[f] = options[f]                              # Set the existing option value
         else
           field = self.get_field(f, d)
           if field[:display] != :ignore
-            if !field[:default].nil?
+            if field[:default]
```

Comment:
here again, ensuring that `field[:default]` does not contain `false`
`present?` may be safer, or `key?`?

(I'd prefer to have it the way you coded it up here)